src/RequestTrait.php

```diff
@@ -48,23 +48,6 @@
     private $uri;
 
     /**
-     * Supported HTTP methods
-     *
-     * @var array
-     */
-    private $validMethods = [
-        'CONNECT',
-        'DELETE',
-        'GET',
-        'HEAD',
-        'OPTIONS',
-        'PATCH',
-        'POST',
-        'PUT',
-        'TRACE',
-    ];
-
-    /**
```

Comment:
Removed the above, as it's no longer necessary; validation is not against this list.